app/services/triannon/ldp_loader.rb

```diff
@@ -56,30 +54,11 @@ def load_bodies
     # load target objects into @ldp_annotation's (our Triannon::AnnotationLdp object) graph
     def load_targets
       @ldp_annotation.target_uris.each { |target_uri|
-        target_obj_path = target_uri.to_s.split(@base_uri + '/').last
+        target_obj_path = target_uri.to_s.split("#{@base_uri}/").last
```

Comment:
I guess you were only changing this line for style, but I'd guess there's an easier-to-understand way to do this.